jira/autokitteh.yaml

```
@@ -0,0 +1,14 @@
+version: v1
+
+project:
+  name: jira_to_calendar_sync
```

Also please rename the directory from "jira" to this project name.

```suggestion
  name: jira_to_google_calendar
```